src/frame.ts

```diff
@@ -489,8 +492,8 @@ export const addElementsToFrame = (
         false,
       );
 
-      const frameIndex = nextElementsIndex[frame.id];
-      const elementIndex = nextElementsIndex[element.id];
+      const frameIndex = nextElementsIndex[frame.id] ?? -1;
+      const elementIndex = nextElementsIndex[element.id] ?? -1;
```

Comment:
@Mrazator added a fallback to -1 to align with previous behavior as it's possible the frame isn't found